easybuild/easyconfigs/j/jax/jax-0.2.19-foss-2020b.eb

```diff
@@ -84,10 +75,8 @@ components = [
         ],
         'start_dir': 'jax-jaxlib-v%s' % local_jaxlib_ver,
         'prebuildopts': local_jax_prebuildopts,
-        'use_pip': True,
         'sanity_pip_check': True,
```

Comment:
This is futile, there's no sanity check for each bundle component individually (like there is for extensions).